eduNEXT/platform-plugin-forum-email-notifier
# feat: implement email digest logic

### Description

This PR creates a model to store the digest of forum updates per user, the logic to store the email digest and update it, and a command to emit the email digest.

### Testing instructions

- Login as an instructor
- Setup the email digest feature as daily
- Login as a student
- Interact with the forum (comment, answer or new thread)
- Verify in the admin that a record is stored in http://local.overhang.io:8000/admin/platform_plugin_forum_email_notifier/forumnotificationdigest/
- Run the following command to trigger the email notification for the digest: `tutor dev exec lms ./manage.py lms forum_digest --digest daily`
- Verify the tasks are created correctly

**Merge checklist:**
Check off if complete *or* not applicable:
